tests/mrcc/rohf_ccsdt_q_/input.dat

```diff
@@ -0,0 +1,34 @@
+memory 500 Mb
```

Comment:
I don't know that we need to set mem in test cases unless it particularly needs more than the default (500 MiB).